assets/eip-5007/README.md

```diff
@@ -0,0 +1,15 @@
+# EIP5007
+This standard is an extension of [ERC-721](./eip-721.md). It proposes some additional functions (`startTime`, `endTime`, `isValidNow`) to help with on-chain time management.
```

Comment:
```suggestion
This standard is an extension of [ERC-721](../EIPS/eip-721.md). It proposes some additional functions (`startTime`, `endTime`, `isValidNow`) to help with on-chain time management.
```

Reply:
Wouldn't it be `../../EIPS/eip-721.md`?  `assets/eip-5007/../EIPS == assets/EIPS` and `/assets/eip-5007/../../EIPS == /EIPS`.

Reply:
> Wouldn't it be `../../EIPS/eip-721.md`? `assets/eip-5007/../EIPS == assets/EIPS` and `/assets/eip-5007/../../EIPS == /EIPS`.

Yes, it should be   `../../EIPS/eip-721.md`.